### PR TITLE
UI - Platform URLs and PR info on Application Details Page

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/22314

*Description of changes:*
**Design**
Host: https://www.figma.com/design/R9GF5wvwq2TrnbnsbhPfvV/STR-Application-Tickets?node-id=4343-4474&t=bKyESWphmXFWWL8I-4
Examiner: https://www.figma.com/design/R9GF5wvwq2TrnbnsbhPfvV/STR-Application-Tickets?node-id=4411-5157&t=bKyESWphmXFWWL8I-4

**PR**
image
image
image
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by shaangill025** [comment]
Typo as seen in the screenshots above has been fixed.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-145-hfd7xjzx.web.app

File: strr-web/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "strr-web",
-  "version": "0.1.24",
```
Comment (rstens) [inline strr-web/package.json:3]: A 2 point jump?

**Review by rstens — COMMENTED** [review]
Labeled the security hotspots as "fixed"

**Comment by dimak1** [comment]
@shaangill025 As a suggestion, could we sanitize the URL before capturing them in DB? Instead of sanitize when presenting them on the Details page.

File: strr-web/package.json (hunk 1 above)
Comment (shaangill025) [reply strr-web/package.json:3]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-145-hfd7xjzx.web.app

**Review by rstens — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-145-hfd7xjzx.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-145-hfd7xjzx.web.app

**Comment by shaangill025** [comment]
@rstens Once this is merged, for QA application ids 72, 69, 68 and 64 may be helpful.

**Review by dimak1 — APPROVED** [review]
Looks good!

**Pull request merged**